### PR TITLE
Sunny sprint 7

### PR DESCRIPTION
**Pull request by xianyi555**
added the validation

### CODE REVIEW

File: app/controllers/site_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class SiteController < ApplicationController
   def index
+  	@cities = City.all
```
Comment (esthersweon) [inline app/controllers/site_controller.rb]: You don't iterate through these cities in `views/site/index.html`, but you should.

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -4,10 +4,10 @@
     <h1>Log In</h1>
     <%= form_for @user, url: "/sessions", method: "post" do |f| %>
         <div class="form-group">
-          <%= f.email_field :email, placeholder: "Email", class: "form-control" %>
+          <%= f.email_field :email, placeholder: "Email", required: true, class: "form-control" %>
         </div>
         <div class="form-group">
-          <%= f.password_field :password, placeholder: "Password", class: "form-control" %>
+          <%= f.password_field :password, placeholder: "Password", required: true, class: "form-control" %>
```
Comment (esthersweon) [inline app/views/sessions/new.html.erb]: You can also validate length of password – please do for all password fields in your templates.

Comment (xianyi555) [reply app/views/sessions/new.html.erb]: Cool! I forgot that. will do now!

Comment (xianyi555) [reply app/views/sessions/new.html.erb]: I just notice that I did the validate length of password at user model.

**Pull request merged**